euphony/build.gradle

```diff
@@ -150,4 +152,5 @@ dependencies {
     androidTestImplementation "androidx.compose.ui:ui-test-junit4:$compose_version"
     androidTestImplementation "org.mockito:mockito-core:4.7.0"
     androidTestImplementation 'com.linkedin.dexmaker:dexmaker-mockito-inline:2.25.1'
+    androidTestImplementation 'androidx.test:core:1.2.0'
```

Comment:
I think you updated `build.gradle` when you implemented `EuRxManager(context)`. So, we don't need to update any line of `build.gradle`.
- we can revert line 67, 140, 155